iterable-ingress-extension/src/main/java/com/mparticle/ext/iterable/IngressQueueManager.java

```diff
@@ -17,12 +24,33 @@ public IngressQueueManager(SqsClient sqsClient, String queueUrl) {
 
   public void enqueueMessage(String message) {
     SendMessageRequest req =
-        SendMessageRequest.builder().queueUrl(queueUrl).messageBody(message).build();
-    sqsClient.sendMessage(req);
+            SendMessageRequest.builder().queueUrl(queueUrl).messageBody(message).build();
```

Comment:
I would split this into multiple lines, like this:
```suggestion
            SendMessageRequest.builder()
            .queueUrl(queueUrl)
            .messageBody(message)
            .build();
```